ic-os/components/setupos-scripts/setup-hostos-config.sh

```diff
@@ -26,6 +26,20 @@ function copy_config_files() {
         log_and_halt_installation_on_error "1" "Configuration file 'config.ini' does not exist."
     fi
 
+    if [ -v FIREWALL_FILE ]; then
```

Comment:
What is the purpose of the FIREWALL_FILE ev? Why don't we just copy over /var/ic/config/firewall.json if it exists?